README.md

```diff
@@ -15,6 +15,18 @@ See https://github.com/databricks/cli/releases for releases and
 [the docs pages](https://docs.databricks.com/dev-tools/cli/databricks-cli.html) for
 installation instructions.
 
+------
+You can use the CLI through a Docker image by pulling from `ghcr.io`. You can find more releases
+at: https://github.com/databricks/cli/pkgs/container/cli.
+```
+docker pull ghcr.io/databricks/cli:latest
+```
+
+Example of how to run the CLI using the docker image. [TODO: Link docs here once live.]:
```

Comment:
This is blocking merge.

Reply:
Yeah, I don't want to merge it before the docker packages are public anyway. Will update once that's taken care of.

Reply:
Added link for this.